spec/system/filter_spec.rb

```diff
@@ -396,7 +395,7 @@
 
         within(selected_govuk_tab) do
           expect(page).to have_content(closed_planning_application.reference)
-          expect(page).to have_content(other_closed_planning_application.reference)
+          # expect(page).to have_content(other_closed_planning_application.reference)
```

Comment:
This and another assertion later are commented out because they're closed and belonging to another user, and waiting to confirm what (if anything) we want to do about that.